src/MooLite/plugins/EquipmentExporter/EquipmentExporterPluginDisplay.vue

```diff
@@ -0,0 +1,162 @@
+<script setup lang="ts">
+import MooDivider from "src/components/atoms/MooDivider.vue";
+import {EquipmentExporterPlugin} from "src/MooLite/plugins/EquipmentExporter/EquipmentExporterPlugin";
+import {computed} from "vue";
+
+const props = defineProps<{
+  plugin: EquipmentExporterPlugin;
+}>();
+
+const equipmentData = computed(() => {
+  return props.plugin.getEquipmentSummary();
+});
+
+const exportDetails = () => {
```

Comment:
This giant block of logic is better suited for the actual plugin, so we can keep the Vue component for presentation logic only.

Reply:
If the plugin gets too big, you are free to create addition helper `.ts` files